soaktest/go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/confluentinc/confluent-kafka-go/soaktest
+
+go 1.13
+
+replace github.com/confluentinc/confluent-kafka-go => ../
+
+require (
+	github.com/DataDog/datadog-go v4.8.3+incompatible
```

Comment:
Outside of the scope of this PR, but be nice to bump some of these deps to drop the `+incompatible` tags. But in a separate PR IMO.

Reply:
Agree, I suggest to do in another PR just to make sure we dont deliver too many changes at once